Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.0-stretch
+FROM golang:latest
```

Comment:
Hmm, I tend to like pinning versions, but I can see the use in tracking latest too…